generators/client/templates/vue/src/test/javascript/spec/app/core/ribbon/ribbon.component.spec.ts.ejs

```diff
@@ -46,25 +42,9 @@ describe('Ribbon', () => {
     expect(ribbon.ribbonEnv).toBeNull();
   });
 
-  it('should not have ribbonEnv when no activeProfiles', async () => {
-    await wrap({ data: {} });
-    expect(ribbon.ribbonEnv).toBeNull();
-  });
-
-  it('should not have ribbonEnv when empty activeProfiles', async () => {
-    await wrap({ data: { activeProfiles: [] } });
-    expect(ribbon.ribbonEnv).toBeNull();
-  });
-
   it('should not have ribbonEnv when empty activeProfiles', async () => {
```

Comment:
Please, can you also fix the test title as it was wrong from the beginning (copy / paste from test above I guess) ?

Reply:
done